### PR TITLE
Group list

### PR DESCRIPTION
**Pull request by 4sum1**
I finished the group list page and add the add_group modal, but modal isn't connected to button yet.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9783,7 +9783,8 @@
             "version": "7.21.0-placeholder-for-preset-env.2",
             "resolved": "https://registry.npmjs.org/@babel/plugin-proposal-private-property-in-object/-/plugin-proposal-private-property-in-object-7.21.0-placeholder-for-preset-env.2.tgz",
```
Comment (Ninit29) [inline package-lock.json]: no need to push package-lock.json

Comment (4sum1) [reply package-lock.json]: > no need to push package-lock.json

How can I change this?

Comment (Ninit29) [reply package-lock.json]: just remove in your PR, delete the file

Comment (4sum1) [reply package-lock.json]: @Ninit29 What does PR stand for?

**Pull request merged**